composer/lib/dependabot/composer/file_parser.rb

```diff
@@ -165,9 +165,18 @@ def git_dependency_details(package_details, requirement)
       end
 
       def lockfile_details(name:, type:)
-        key = lockfile_key(type)
-        parsed_lockfile.fetch(key, []).find { |d| d["name"] == name }
-      end
+		begin
```

Comment:
seems the whitespace formatting can be aligned?

Reply:
yeah, I did an edit in a code editor that has its indent settings configured wrong.  I will fix the cosmetics.